api_tests/dry/rfc003_swap_reject.ts

```diff
@@ -313,55 +318,54 @@ describe("SWAP request REJECTED", () => {
 
     it("[Bob] Should be in the Rejected State after declining a swap request providing a reason", async function() {
         await bob.poll_comit_node_until(
-            chai,
             bob_stingy_swap_href,
-            body => body.state.communication.status === "REJECTED"
+            (body: SwapResponse) =>
+                body.state.communication.status === "REJECTED"
         );
     });
 
     it("[Alice] Should be in the Rejected State after Bob declines a swap request providing a reason", async () => {
         await alice.poll_comit_node_until(
-            chai,
             alice_stingy_swap_href,
-            body => body.state.communication.status === "REJECTED"
+            (body: SwapResponse) =>
+                body.state.communication.status === "REJECTED"
         );
     });
 
     it("[Bob] Can execute a decline action, without providing a reason", async () => {
-        let bob_decline_href_2;
+        let bob_decline_href_2: string;
 
         await chai
             .request(bob.comit_node_url())
             .get(bob_reasonable_swap_href)
-            .then(res => {
+            .then(async function(res) {
```

Comment:
I really dislike the way we mix `async/await` and `.then` in these tests. Since these things are going away, there is no point in changing it now but in your other PR, you could think about changing everything to `async/await` to make things more readable and consistent.

Reply:
I was trying to adopt a more functional style.

You reckon all `async/await` is a good way to go?

Reply:
I would prefer that yes.

Reply:
Ok, I'll do that in the other PR then.